systemtests/src/main/java/io/enmasse/systemtest/platform/apps/SystemtestsKubernetesApps.java

```diff
@@ -156,14 +156,21 @@
         };
 
         POSTGRESQL_EXAMPLE_BASE = Paths.get("../templates/iot/examples/postgresql/deploy");
-        var pgBase = Paths.get("../templates/iot/examples/postgresql/create.sql");
+        var pgDevCon = Paths.get("../templates/iot/examples/postgresql/create.devcon.sql");
```

Comment:
I think these should use Environment.getInstance().getTemplatesPath() so that they can work with any release bundle.

Reply:
Done.